forte/sparse_ci/sq_operator_string.h

```diff
@@ -91,11 +95,18 @@ class SQOperatorString {
     Determinant& cre_mod();
     /// @return a Determinant object that represents the annihilation operators
     Determinant& ann_mod();
-    /// @return true if this operator is a number operator (i.e. it contains no creation or
-    /// annihilation  operators)
-    bool is_number() const;
+    /// @return a op_tuple_t that represents the operator
+    op_tuple_t op_tuple() const;
+    /// @return the number component of this operator
```

Comment:
Please document what the "number component" is. This is non-standard terminology, and it take me a minute of staring at the implementation to figure it out.

Reply:
Good point!